src/sp_var_value.c

```diff
@@ -71,7 +71,7 @@ static zval *get_var_value(zend_execute_data *ed, const char *var_name,
   if (is_param) {
     zval *zvalue = get_param_var(ed, var_name);
     if (!zvalue) {
-      return get_local_var(ed, var_name);
+      return NULL;
```

Comment:
It seems that this change, while welcome, is unrelated to this feature, isn't it?
Also, it would be nice to have a test for it.

Reply:
This part was defaulting on getting a local variable when the parameter didn't exists, so it is kind of related, and I couldn't put a warning there because we don't have access to the name of the function.
It seems that it isn't used anywhere else, but the test suite is broken on alpine on master, with 107 tests failing.

Reply:
Feel free to fix the testsuite in an other commit ♥

Reply:
I wanted to fix the CI in another pull request.

Reply:
Yeah, I think that adding back distributions to the CI is a good idea